src/components/AddressVerificationContainer/index.tsx

```diff
@@ -236,8 +230,10 @@ function AddressVerificationContainer() {
           )}
 
           {workAddress?.length > 0 && (
-            <Box sx={styles.workAddressContainer}>
-              <PageSubHeading>{t("verifyAddress.workAddress")}</PageSubHeading>
+            <Box mt="l">
+              <PageSubHeading id="verify-work-address-heading" mb="s">
+                {t("verifyAddress.workAddress")}
```

Comment:
This is still be rendered as `Work` on Step 5. This should instead be rendered as `Alternate`.